packages/module/patternfly-docs/content/examples/catalogTile.css

```diff
@@ -21,24 +17,21 @@
   color: var(--pf-t--global--text--color--link--hover);
   text-decoration: underline;
 }
-.ws-react-e-catalog-tile .pf-v6-c-card__actions {
-  padding-left: 5px;
-}
 .ws-react-e-catalog-tile .catalog-tile-pf-icon {
-  font-size: 40px;
-  height: 40px;
+  font-size: 37px;
```

Comment:
Updated to 37px per figma, but I don't know that it really matters if that needs to change to something else?